auth/index.js

```diff
@@ -0,0 +1,28 @@
+import jwt from 'jsonwebtoken';
+import debug from 'debug';
+
+import { AuthorizationError } from '../graphql/errors';
+
+const log = debug('fcc:auth');
+const { JWT_CERT, NODE_ENV } = process.env;
+
+export function verifyWebToken(ctx) {
+  log('Verifying token');
+  const token = ctx && ctx.headers && ctx.headers.authorization;
+  if (!token) {
+    throw new AuthorizationError({
+      message: 'You must supply a JSON Web Token for authorization!'
+    });
+  }
+  let decoded = null;
+  let error = null;
+  try {
+    decoded = jwt.verify(token.replace('Bearer ', ''), JWT_CERT, {
+      ignoreExpiration: NODE_ENV === 'test'
```

Comment:
`NODE_ENV === 'test'` => `NODE_ENV === 'development'`

or even better

`NODE_ENV !== 'production'`

Having it as 'test' which is a non-standard value will make the express server crash. This is because the third-party deps we use, such as playground rely on `NODE_ENV`

Mutating it to a non-standard value a bad idea IMHO.